wandb/sdk/wandb_init.py

```diff
@@ -85,13 +87,10 @@ def __init__(self):
         self._wl = None
         self._reporter = None
         self._use_sagemaker = None
```

Comment:
this one can also be folded into the `_init_telemetry_obj` solution

Reply:
oh,, thanks. i missed that one, fixing.